internal/pkg/agent/transpiler/vars_test.go

```diff
@@ -185,13 +191,10 @@ func TestVars_Replace(t *testing.T) {
 			false,
 		},
 		{
-			`list inside string ${un-der_score.list} causes no match`,
-			NewList([]Node{
-				NewStrVal("array1"),
-				NewStrVal("array2"),
-			}),
+			`list inside string ${un-der_score.list} strings array`,
+			NewStrVal(`list inside string [array1,array2] strings array`),
+			false,
```

Comment:
Why is this matching now? It doesn't seem like the input has changed.

Reply:
This test was actually broken and was never correct.